util.go

```diff
@@ -7,21 +7,20 @@ import (
 	"errors"
 	"io"
 	"io/ioutil"
-	"log"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
-func ChecksumFile(file string) string {
+func ChecksumFile(file string) (string, error) {
 	hasher := sha256.New()
 	s, err := ioutil.ReadFile(file)
```

Comment:
Wasn't ioutil supposed to be removed?

Reply:
Yes, but I thought I'd leave them alone since you had sent a pull-request for it :) I haven't merged that working tree towards master.

Reply:
Well my pull request was against master. I will have to update it once this gets merged or rebase it against this branch.

Reply:
I'll do the merging work :) No need to worry about it.